mmv1/products/gkehub/api.yaml

```diff
@@ -75,8 +77,9 @@ objects:
           The unique identifier of the membership.
       - !ruby/object:Api::Type::String
         name: 'description'
+        min_version: beta
```

Comment:
Ah, beta-only makes so much more sense for this field!

We should also mark this as deprecated, as it is on the way out rather than a new field. We should be able to do that by adding:
`deprecation_message: some message here`  to api.yaml

Reply:
Fixed.